icu4j/maven-build/local-maven-repo/com/ibm/icu/icudata/73.1-SNAPSHOT/icudata-73.1-SNAPSHOT.pom

```diff
@@ -0,0 +1,13 @@
+<!--
+© 2022 and later: Unicode, Inc. and others.
+License & terms of use: http://www.unicode.org/copyright.html
+-->
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
When building you will see a warning:
```
Warning:  The POM for com.ibm.icu:icudata:jar:73.1-SNAPSHOT is invalid, transitive dependencies (if any) will not be available, enable debug logging for more details
```

That is caused by the comment being before the  processing directive (`<?xml ...>`).
That should be the first line in the file.

Go in the `icu4j-maven-build` folder and do a 
```
grep -r -n "<?xml" .
```

Fix all instances that show a line number different than 1 (by moving the directive before the copyright comment)
There are 9 files with this problem, 6 `.xml` and 3 `.pom`

Reply:
Take a look at https://github.com/google/addlicense

It is a Google tool, and knows how to add license in the right place and with the proper comment style for all kind of formats, including .xml. And it does add the copyright after the directive.